NetProbe/View/SettingsTableViewController.m

```diff
@@ -58,11 +59,18 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         cell.accessoryView = switchview;
     }
     else {
+        cell = [tableView dequeueReusableCellWithIdentifier:@"CellSub" forIndexPath:indexPath];
+        NSString *current = [settingsItems objectAtIndex:indexPath.row];
+        cell.textLabel.text = NSLocalizedString(current, nil);
+        cell.detailTextLabel.text = [[NSUserDefaults standardUserDefaults] objectForKey:current];
+    }
+    /*
+    else {
```

Comment:
Yes, I want to keep this until we decide how to finally implement settings. At the moment I removed the Advanced settings screen as we have only three items